app/components/meeting-form.hbs

```diff
@@ -92,7 +92,10 @@
   <div class="au-c-meeting-chrome">
     <div class="au-c-meeting-chrome__paper">
       {{!-- Meeting title --}}
-      <AuHeading>{{t "meetingForm.meetingHeadingPartOne"}} <span class="au-c-meeting-chrome__highlight">{{@zitting.bestuursorgaan.isTijdsspecialisatieVan.naam}}</span>, {{t "meetingForm.meetingHeadingPartTwo"}} <span class="au-c-meeting-chrome__highlight">{{moment-format  @zitting.geplandeStart "DD/MM/YYYY HH:mm"}}</span></AuHeading>
+      <AuHeading>
+        {{t "meetingForm.meetingHeadingPartOne"}} <span class="au-c-meeting-chrome__highlight">{{@zitting.bestuursorgaan.isTijdsspecialisatieVan.naam}},</span>
```

Comment:
I moved the `,` to be inside the highlight because if the name is long enough to wrap multiple lines, the comma floats on the line below, which looks funky:

![image](https://user-images.githubusercontent.com/3533236/120005196-ea9d8e00-bfd7-11eb-9d64-c5c1e956e6c7.png)